tasks/src/tabular-regression/about.md

```diff
@@ -3,7 +3,21 @@
 Tabular regression is the task of predicting a numerical value given a set of attributes/features. *Tabular* meaning that data is stored in a table (like an excel sheet), and each sample is contained in its own row. The features used to predict our target can be both numerical and categorical. However, including categorical features often requires additional preprocessing/feature engineering (a few models do accept categorical features directly, like [CatBoost](https://catboost.ai/)). An example of tabular regression would be predicting the weight of a fish given its' species and length.
 
 ## Use Cases
+Tabular data is one of the most prevalent data out there, and many raw-datasets once we extract features from them can be arrange in tabular form. Below you can find some of the popular tabular use-cases:-
+- [Predicting Air Pollution](https://www.kaggle.com/competitions/tabular-playground-series-jun-2021) 
```

Comment:
Can you write explanations on these instead of giving links in kaggle?

Reply:
Updated the changes [here](https://github.com/huggingface/hub-docs/pull/434/commits/b45cc5850619518d70f17653abb205996ae5a907).